internal/transport/http2_client_test.go

```diff
@@ -27,6 +27,11 @@ import (
 	"google.golang.org/grpc/resolver"
 )
 
+const IntendedTestCaseString = "This is a test case string"
+
+// ClientPreface is the HTTP/2 client preface string.
+var ClientPreface = []byte("PRI * HTTP/2.0\r\n\r\nSM\r\n\r\n")
```

Comment:
For ClientPreface assignment you can do simply this since the string is already defined https://github.com/grpc/grpc-go/blob/ced812e3287e15a009eab5b271c25750050a2f82/internal/transport/http_util.go#L50

Reply:
change done